File: jdt-dpgen-cli/src/main/resources/dpgen-output/src/main/scala/com/gof/creational/builder/EmployeeBuilder.scala
package com.gof.creational.builder

import java.util.List
//remove if not needed
import scala.collection.JavaConversions._

class EmployeeBuilder extends Builder {

  var employee: Employee = _

  def buildAge(age: Int): Builder = {
    employee.setAge(age)
    this
  }

  def buildLastName(lastName: String): Builder = {
    employee.setLastName(lastName)
    this
  }

  def buildFirstName(firstName: String): Builder = {
    employee.setFirstName(firstName)
    this
  }

  def buildResponsibilities(responsibilities: List[String]): Builder = {
    employee.setResponsibilities(responsibilities)
    this
  }
}

File: jdt-dpgen-cli/src/main/resources/dpgen-output/src/main/java/com/gof/creational/builder/EmployeeBuilder.java
package com.gof.creational.builder;

import java.util.List;

public class EmployeeBuilder extends Builder {

	public Employee employee;

	public Builder buildAge(int age) {
		employee.setAge(age);
		return this;
	}

	public Builder buildLastName(String lastName) {
		employee.setLastName(lastName);
		return this;
	}

	public Builder buildFirstName(String firstName) {
		employee.setFirstName(firstName);
		return this;
	}

	public Builder buildResponsibilities(List<String> responsibilities) {
		employee.setResponsibilities(responsibilities);
		return this;
	}
}